build.rs

```diff
@@ -4,7 +4,7 @@ const SRC_LIB_RS_CONTENTS: &str = include_str!("src/lib.rs");
 const EXPECTED_SRC_LIB_RS_PREFIX: &str = "\
 //! Port of LLVM's APFloat software floating-point implementation from the
 //! following C++ sources (please update commit hash when backporting):
-//! https://github.com/llvm/llvm-project/commit/";
+//! <https://github.com/llvm/llvm-project/commit/";
```

Comment:
```suggestion
//! <https://github.com/llvm/llvm-project/commit/>";
```

Reply:
Thanks for the look - the build script expects a hash as part of the link directly after that. The closing `>` gets matched here https://github.com/rust-lang/rustc_apfloat/pull/15/files#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R23 😆